flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/util/TableTestBase.scala

```diff
@@ -810,13 +811,10 @@ case class JavaBatchTableTestUtil(test: TableTestBase) extends JavaTableTestUtil
 /**
   * Batch/Stream [[org.apache.flink.table.sources.TableSource]] for testing.
   */
-class TestTableSource(isBatch: Boolean, schema: TableSchema)
+class TestTableSource(val isBounded: Boolean, schema: TableSchema)
```

Comment:
nit: the previous code was more readable.

Reply:
bounded `TestTableSource` will also used in streaming test, so i think isBounded is more suitable.